test/pls.jl

```diff
@@ -94,6 +99,7 @@ fits = Dict(k => fit.(MixedModel, forms[k], Ref(MixedModels.dataset[k])) for k i
     @test fm1.objective == objective(fm1)
     @test fm1.σ ≈ 49.510099986291145 atol=1.e-5
     @test fm1.X == ones(30,1)
+    ds = MixedModels.dataset(:dyestuff)
```

Comment:
since you used fancy `using`:

```suggestion
    ds = dataset(:dyestuff)
```